pubkey/option_test.go

```diff
@@ -21,102 +21,71 @@ import (
 	"reflect"
 	"testing"
 	"time"
+
+	urlutil "github.com/yahoojapan/athenz-authorizer/internal"
```

Comment:
same as above, rename it to url

Reply:
https://github.com/yahoojapan/athenz-authorizer/blob/bf4a948c7cbfd3aee6422d75801425ad43a5aac7/jwk/option.go#L38-L40

name conflict with the `url String`, I think rename during import is better...

Reply:
But the standard url package is not using in this file, so I think it is ok to change to url.
If later we need it we can rename the current one or add a alias to the standard url package inside this file...

Reply:
Even though I think rename the variable name to `u` instead of `url` is better,,, I think it is fine for now.